package.json

```diff
@@ -31,10 +31,12 @@
   },
   "dependencies": {
     "body": "^5.0.0",
+    "commander": "^2.6.0",
```

Comment:
I like the idea of pruning the dependencies to production, I don't really like to have commander now as a dependency. This is only used when ringpop is ran during testing right? Is there a way to have separate testing dependencies separate from `devDependencies`